src/components/pages/Offers/Offer/Stocks/StockItem/domain.js

```diff
@@ -93,10 +93,11 @@ export const hasStockBeenUpdated = (originalStock, updatedStock) => {
     )
   }
 
+  const momentOriginalBeginningDatetime = moment(originalStock.beginningDatetime)
   const momentBookingLimitDatetime = moment(originalStock.bookingLimitDatetime)
 
   return !(
-    originalStock.beginningDatetime === updatedStock.beginningDatetime &&
+    momentOriginalBeginningDatetime.isSame(updatedStock.beginningDatetime, 'minute') &&
```

Comment:
Je pense qu'on peut éviter de déclarer une variable qui n'apporte pas grand chose dazns ce contexte (elle n'est même pas réutilisée) : 
```suggestion
    moment(originalStock.beginningDatetime).isSame(updatedStock.beginningDatetime, 'minute') &&
```
Du coup tu changerais l'autre aussi